iOS/3D Cave Runner Xcode/Classes/UI/SplashScreen.mm

```diff
@@ -45,84 +46,76 @@
     bool rotateToPortrait  = _canRotateToPortrait || _canRotateToPortraitUpsideDown;
     bool rotateToLandscape = _canRotateToLandscapeLeft || _canRotateToLandscapeRight;
 
-    if (_isOrientable)
-    {
-        if (orientPortrait && rotateToPortrait)
-            return "-Portrait";
-        else if (orientLandscape && rotateToLandscape)
-            return "-Landscape";
-        else if (rotateToPortrait)
-            return "-Portrait";
-        else
-            return "-Landscape";
-    }
-    return "";
```

Comment:
why don't you need to check if it is orientable anymore? what happen if the game is force to always be a specific orientation?